.github/CODEOWNERS

```diff
@@ -6,7 +6,7 @@
 #/avm/res/aad/domain-service/ @Azure/avm-res-aad-domainservice-module-owners-bicep
 /avm/res/analysis-services/server/ @Azure/avm-res-analysisservices-server-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/api-management/service/ @Azure/avm-res-apimanagement-service-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/app/container-app/ @Azure/avm-res-app-containerapp-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/app/container-app/ @Azure/avm-res-app-containerapp-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Please note, that the team must also be created as per: https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships (including the team's parent-child) relationship

Reply:
Just finished, thanks for the reminder!

Reply:
Unresolving to merge only after the relationship was approved by the parent team's owner

Reply:
Hey @oZakari, as codeowners file shows errors, could you check these guidelines https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships ? It's mainly about creating relevant Azure teams and requesting them to be added to relevant parent teams. Let us know if you get stuck in the approval process so we can try to unblock it.